### ISSUE TITLE
Sulla stuck at Authenticating

### ISSUE THREAD

**Issue by foomoto** [danielcardeenas/sulla#386]
Recently, I noticed sulla nolonger injects API. This is by far the cleanest repo on this problem. Would like to support, but can Daniel or anyone give me a walk through.

**Comment by MohakChugh** [comment]
I have been encountering the same error. It seems to be logged in from the phone side, so the issue seems from the library itself.

**Comment by wellcoj** [comment]
Same problem here since last friday. Any solution?

**Comment by marutichintan** [comment]
Same Problem here, Some thing is change, because some update in script other WAPI injection script is also not working.

**Comment by s2click** [comment]
Hello, sulla is no longer updated, they migrated to the venom repo, where it is 100% working, follow the repo link: https://github.com/orkestral/venom

**Comment by XNexusPrimeX** [comment]
venom also doesn't work, he has the same problem

**Comment by orkestral** [comment]
> venom also doesn't work, he has the same problem

Venom no is with problem

**Comment by pablocorezzola** [comment]
> > venom also doesn't work, he has the same problem
> 
> Venom is not in trouble

venom does not start

**Comment by wallace-araujo** [comment]
venom .it's working perfect

**Comment by foomoto** [comment]
yes its working well.

**Comment by panzi** [comment]
I will try out venom tomorrow myself, but if it's not working for some, but is working for others, maybe WhatsApp us A/B testing a new web interface? Is there a way to debug that?

**Comment by authmane512** [comment]
It doesn't work for me too. it said "couldn't scan code. make sure you're on web.whatsapp.com"

**Comment by panzi** [comment]
Yes, venom-bot doesn't work either for me. Like sulla it gets to `- Authenticate to continue`, shows the QR code, I scan the QR code and then nothing more happens.

**Comment by orkestral** [comment]
install for npm install venom-bot, is working

**Comment by panzi** [comment]
It's not. See my bug report there.

**Comment by pablocorezzola** [comment]
> It's not. See my bug report there: [orkestral/venom#21](https://github.com/orkestral/venom/issues/21)

check this project https://github.com/pedroslopez/whatsapp-web.js.

**Comment by orkestral** [comment]
this repository is no longer supported, is out of date, has been migrated to what is now functional:
[Venom-Bot](https://github.com/orkestral/venom)

Please leave us a star, there too, encourages us to keep you updated
